spec.html

```diff
@@ -903,27 +903,29 @@ <h1>Static Semantics</h1>
-      <p>In general, when this specification refers to a numerical value, such as in the phrase, "the length of _y_" or "the integer represented by the four hexadecimal digits ...", without explicitly specifying a numeric kind, the phrase refers to a Number. Phrases which refer to a mathematical value are explicitly annotated as such; for example, "the mathematical value of the number of code points in ...".</p>
-      <p>It is not defined to mix Numbers and mathematical values in either arithmetic or comparison operations, and any such undefined operation would be an editorial error in this specification text.</p>
-      <p>The Number value 0, alternatively written 0<sub>𝔽</sub>, is defined as the double-precision floating point positive zero value. In certain contexts, it may also be written as *+0* for clarity.</p>
+      <p>In the language of this specification, numerical values and operations (including addition, subtraction, negation, multiplication, division, and comparison) are distinguished among different numeric kinds using subscripts. The subscript <sub><dfn id="𝔽">𝔽</dfn></sub> refers to Numbers, the subscript <sub><dfn id="ℝ">ℝ</dfn></sub> refers to mathematical values, and the subscript <sub><dfn id="ℤ">ℤ</dfn></sub> refers to BigInts. A subscript is used following each numeric value and operation.</p>
+      <p>For brevity, the <sub>ℝ</sub> subscript can be omitted on numeric values&mdash;a numeric value with no subscript is interpreted to be a mathematical value. An operation with no subscript is interpreted to be a mathematical value operation, unless one of the parameters has a particular subscript, in which case the operation adopts that subscript. For example, 1<sub>𝔽</sub> + 2<sub>𝔽</sub> = 3<sub>𝔽</sub> is a statement about Number values, and 1 + 2 = 3 is a statement about mathematical values.</p>
+      <p>In general, when this specification refers to a numerical value, such as in the phrase, "the length of _y_" or "the integer represented by the four hexadecimal digits ...", without explicitly specifying a numeric kind, the phrase refers to a mathematical value. Phrases which refer to a Number or a BigInt value are explicitly annotated as such; for example, "the Number value of the number of code points in ..." or "the BigInt value of ...".</p>
+      <p>It is not defined to mix Numbers, BigInts and mathematical values in either arithmetic or comparison operations, and any such undefined operation would be an editorial error in this specification text.</p>
```

Comment:
I like this, I would just word it slightly differently.

```suggestion
      <p>Arithmetic or comparison operators applied to mixed-type operands (such as a Number and a mathematical value) are undefined and should be considered an editorial error in this specification.</p>
```

We could also merge it into the above paragraph.